packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -107,10 +114,9 @@ lemmas=test/kontrol/pausability-lemmas.md
 base_module=PAUSABILITY-LEMMAS
 module=OptimismPortalKontrol:$base_module
 rekompile=--rekompile
-rekompile=
+# rekompile=
```

Comment:
Is there a reason to include this line at all?

Reply:
Just for convenience. As described above, unsetting the variable will remove that option, so the comment is there to remove two characters instead of writing the whole thing. I'm not opposed to remove this if you don't like it!
Also note that we now support toml specifications for this options, so in the future all of this might be just contained in the `kontrol.toml` file

Reply:
Cool, seems fine to me!